Resources/Prototypes/_Midnight/Catalog/Fills/Items/paper.yml

```
@@ -0,0 +1,19 @@
+      - id: PaperHoldoutBriefcase
+      - id: PaperHoldoutHat
+      - id: PaperHoldoutShoes
+      - id: PaperHoldoutCoat
+
+- type: entity
+  parent: BoxFolderClipboard
+  id: BoxFolderClipboardSmugglerFilled
+  components:
+  - type: StorageFill
+    contents:
+      - id: ForensicReportPaperSmuggler
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)